packages/api/cms-api/src/dam/files/dto/update-dam-file.args.ts

```diff
@@ -0,0 +1,19 @@
+import { ArgsType, Field, ID } from "@nestjs/graphql";
+import { Type } from "class-transformer";
+import { IsString, ValidateNested } from "class-validator";
+
+import { HasValidFilename } from "../../common/decorators/has-valid-filename.decorator";
+import { UpdateFileInput } from "./file.input";
+
+@ArgsType()
+export class UpdateDamFileArgs {
+    @Field(() => ID)
+    @IsString()
```

Comment:
We could now even add IsUUID

Reply:
3c63f811bfd8bfef19d5577a95c665f002ffcd42